src/containers/EnterpriseApp/EnterpriseApp.test.jsx

```diff
@@ -18,6 +18,7 @@ import { features } from '../../config';
 import NotFoundPage from '../../components/NotFoundPage';
 import { EnterpriseSubsidiesContext } from '../../components/EnterpriseSubsidiesContext';
 
+// eslint-disable-next-line react/function-component-definition
```

Comment:
nit: Why is this rule disabled here but in other test files,  you've replaced arrow function components with `function`?

Reply:
Replacing it here results in following error,
`The module factory of `jest.mock()` is not allowed to reference any out-of-scope variables.`
I have now moved it inside the scope to resolve this issue